drf_integrations/forms.py

```diff
@@ -1,12 +1,10 @@
 import copy
 from django import forms
+from django.contrib.postgres.forms import JSONField
 
 from drf_integrations import models
-from drf_integrations.fields import get_json_field
 from drf_integrations.integrations import Registry
 
-JSONField = get_json_field()
```

Comment:
This goes to the setting `DB_BACKEND_JSON_FIELD` to and imports the class. That setting is set to `django.contrib.postgres.fields.JSONField` on Platform which is for models and not forms.